net/netifyd/patches/01-nd-addr.patch

```diff
@@ -0,0 +1,522 @@
+diff --git a/include/nd-addr.h b/include/nd-addr.h
```

Comment:
You are now adding patch.  
- Why it is required? 
- Why it is not included in upstream? 
   - Where it comes? 
- Is there any pending pull request? 
- Why we should have it?
- Who is author?

And so many other questions, which I can think of.

Reply:
The patch was added as a misguided attempt to get a build through the CI system.

Changes have been merged into v4.4.1 and resubmitted.  Patch removed.